packages/standard-linter/index.js

```diff
@@ -6,29 +6,74 @@ module.exports = {
     "airbnb-typescript/base",
     "prettier",
   ],
-  plugins: ["simple-import-sort", "prettier"],
+  plugins: ["simple-import-sort", "check-file", "unused-imports", "prettier"],
   parser: "@typescript-eslint/parser",
   rules: {
     "@typescript-eslint/no-floating-promises": "error",
-    "@typescript-eslint/no-unused-vars": [
+    "no-console": "error",
```

Comment:
should this always be disallowed by default? this will block testing for sdk since eslint will always be checked before compilation

Reply:
hmm, i think this should be `error` by default. we can still run the app but not commit.

Reply:
Yeah this should just be warn

Reply:
`warn` will still pass lint check and CI.

Reply:
ideally, we should not allow any console log in production, whether server or sdk. we could potentially log sensitive info. thats why we should prevent it in the lint check and CI

Reply:
okay we can do this here. if the project requires console log during development we can override it